crates/header-translator/src/stmt.rs

```diff
@@ -879,32 +882,30 @@ impl fmt::Display for Stmt {
             } => {
                 // TODO: Use ty.get_objc_protocol_declarations()
 
-                let macro_name = if ty.generics.is_empty() {
+                let macro_name = if generics.is_empty() {
                     "extern_class"
                 } else {
                     "__inner_extern_class"
                 };
 
                 writeln!(f, "{macro_name}!(")?;
```

Comment:
Do we need to emit the feature attributes here too (and at line 922 below)?

```rust
if let Some(feature) = id.feature() {
    writeln!(f, "    #[cfg(feature = \"{feature}\")]")?;
}
```

This way we generate output like this:

```rust
extern_class!(
    #[cfg(feature = "WebKit_DOMAbstractView")]
    #[derive(Debug, PartialEq, Eq, Hash)]
    pub struct DOMAbstractView;

    #[cfg(feature = "WebKit_DOMAbstractView")]
    unsafe impl ClassType for DOMAbstractView {
        #[inherits(WebScriptObject, NSObject)]
        type Super = DOMObject;
    }
);
```

Reply:
True, this is now done